proof/refine/ARM_HYP/ArchAcc_R.thy

```diff
@@ -1683,5 +1690,166 @@ lemma dmo_clearMemory_invs'[wp]:
   apply fastforce
   done
 
+lemma pspace_aligned_cross:
+  "\<lbrakk> pspace_aligned s; pspace_relation (kheap s) (ksPSpace s') \<rbrakk> \<Longrightarrow> pspace_aligned' s'"
```

Comment:
Technically speaking, these lemmas should probably not be in an Arch theory, but I think we can sort that out in the arch split. The existing ones are not in consistent places either.